package.json

```diff
@@ -118,7 +118,8 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "pretty-quick --staged"
+      "pre-commit": "pretty-quick --staged",
+      "pre-push": "git branch | grep \"*\" | egrep -v \"^* develop$\" && git branch | grep \"*\" | egrep -v \"^* master$\""
```

Comment:
this will prevent accidently pushing to master or develop by any one

Reply:
nice